stacks/container-apps/augury-alarms.yml

```diff
@@ -0,0 +1,46 @@
+# stacks/container-apps/augury-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Augury alarms
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AuguryWorkerLogGroupName:
+    Type: String
+Resources:
+  AuguryActualsIngestJobInvocationMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '[ImpressionsFetchJob] Success'
+      LogGroupName: !Ref AuguryWorkerLogGroupName
+      MetricTransformations:
+        - MetricName: !Sub "augury_actuals_rollups_ct_${EnvironmentTypeAbbreviation}"
```

Comment:
If these metrics get moved to their own namespace, maybe the metric name can be tweaked a bit. Also, by convention, CloudWatch metrics are named using PascalCase; the Dovetail ones are a little unusual in that regard.

Reply:
Any particular tweaking to the name? Maybe add the namespace as a prefix?